contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -0,0 +1,933 @@
+     * that NFT LP position to a dead address (transfer to zero address not allowed.) See example of such
+     * NFT LP token:
+     * https://basescan.org/token/0x827922686190790b37229fd06084350e74485b72?a=413296#inventory
+     */
+
+    /***************************************
+            Storage slot members
+    ****************************************/
+
+    /// @notice tokenId of the liquidity position
+    uint256 public tokenId;
+    /// @dev Minimum amount of tokens the strategy would be able to withdraw from the pool.
+    ///      minimum amount of tokens are withdrawn at a 1:1 price
+    uint256 public underlyingAssets;
+    /**
+     * @notice Specifies WETH to OETHb ratio the strategy contract aims for after rebalancing
+     * as 18 decimal point
+     *
+     * e.g. 0.2e18 means 20% WETH 80% OETHb
+     */
+    uint256 public poolWethShare;
+    /**
+     * @notice Specifies how the target WETH share of the pool defined by the `poolWethShare` can
+     * vary from the configured value after rebalancing. Expressed as 18 decimal point
+     */
+    uint256 public poolWethShareVarianceAllowed;
+    /// @dev reserved for inheritance
+    int256[46] private __reserved;
+
+    /***************************************
+          Constants, structs and events
+    ****************************************/
+
+    /// @notice The address of the Wrapped ETH (WETH) token contract
+    address public immutable WETH;
+    /// @notice The address of the OETHb token contract
+    address public immutable OETHb;
+    /// @notice lower tick set to -1 representing the price of 1.0001 of WETH for 1 OETHb.
+    int24 public immutable lowerTick;
+    /// @notice lower tick set to 0 representing the price of 1.0000 of WETH for 1 OETHb.
+    int24 public immutable upperTick;
+    /// @notice tick spacing of the pool (set to 1)
+    int24 public immutable tickSpacing;
+    /// @notice the swapRouter for performing swaps
+    ISwapRouter public immutable swapRouter;
+    /// @notice the underlying AMO Slipstream pool
+    ICLPool public immutable clPool;
+    /// @notice the gauge for the corresponding Slipstream pool (clPool)
+    /// @dev can become an immutable once the gauge is created on the base main-net
+    ICLGauge public immutable clGauge;
+    /// @notice the Position manager contract that is used to manage the pool's position
+    INonfungiblePositionManager public immutable positionManager;
+    /// @notice helper contract for liquidity and ticker math
+    ISugarHelper public immutable helper;
+    /// @notice sqrtRatioX96Tick0
+    /// @dev tick 0 has value -1 and represents the lowest price of WETH priced in OETHb. Meaning the pool
+    /// offers less than 1 OETHb for 1 WETH. In other terms to get 1 OETHB the swap needs to offer 1.0001 WETH
+    /// this is where purchasing OETHb with WETH within the liquidity position is most expensive
+    uint160 public immutable sqrtRatioX96Tick0;
```

Comment:
⚪️ Would it be better if these were called `sqrtRatioX96TickLower` and `sqrtRatioX96TickUpper`, since they are built out of `lowerTick` and `upperTick` immutables?

Reply:
good point, renamed: https://github.com/OriginProtocol/origin-dollar/pull/2184/commits/b3abb0d5604fa2a81d1a68a051c6e3e2e0160a78